main/pipeline/pipeline.h

```diff
@@ -7,6 +7,7 @@
 #include "common/kvstore/KeyValueStore.h"
 #include "core/FileHash.h"
 #include "main/options/options.h"
+#include "parser/prism/Parser.h"
```

Comment:
Ditto this include being something in pipeline that we don't want. You'll notice that not a single other include from Sorbet's pipeline shows up here: there are no includes of parser, desugar, rewriter, namer, resolver, cfg, or infer here.

That strongly suggests that we should not also include `parser` here, let alone `parser/prism`.